tests/telemetry/test_writer.py

```diff
@@ -296,12 +296,13 @@ def test_app_started_event_configuration_override(test_agent_session, run_python
 
     # DD_TRACE_AGENT_URL in gitlab is different from CI, to keep things simple we will
     # skip validating this config
-    configurations = test_agent_session.get_configurations(ignores=["DD_TRACE_AGENT_URL"])
+    configurations = test_agent_session.get_configurations(
+        ignores=["DD_TRACE_AGENT_URL", "DD_AGENT_PORT", "DD_TRACE_AGENT_PORT"], remove_seq_id=True, effective=True
```

Comment:
Is this to avoid breaking gitlab/CI?

Reply:
Yup, these configurations are set on ci and can be different from what's run locally. So we just ignore these configs in our test case.